SwrveConversationSDK/Conversation/SwrveContentHTML.m

```diff
@@ -33,18 +33,17 @@ -(id) initWithTag:(NSString *)tag andDictionary:(NSDictionary *)dict {
 }
 
 - (void)webViewDidFinishLoad:(UIWebView *)webView {
-#pragma unused (webView)
     // Set real width
     CGRect frame = _view.frame;
     frame.size.width = _containerView.frame.size.width;
     _view.frame = frame;
     // Measure height
-    NSString *output = [(UIWebView*)_view
-                        stringByEvaluatingJavaScriptFromString:
-                        @"document.height;"];
     frame = _view.frame;
-    frame.size.height = [output floatValue];
+    
+    NSString *scrollHeight = [webView stringByEvaluatingJavaScriptFromString:@"document.body.scrollHeight;"];
+    frame.size.height = [scrollHeight floatValue];
     _view.frame = frame;
+    
```

Comment:
frame = _view.frame
_view.frame = frame
frame = _view.frame
_view.frame = frame

😕 

Can that be cleaned a bit, mildly headache-y the way it is now.